lib/tests/streamlit/runtime/caching/cache_errors_test.py

```diff
@@ -104,20 +112,21 @@ def unserializable_return_value_func():
         self.assertEqual(ep.message_is_markdown, True)
         self.assertEqual(ep.is_warning, False)
 
-    @parameterized.expand(UNEVALUATED_DATAFRAME_TYPES)
-    def test_unevaluated_dataframe_error(self, type_name):
-        if "snowpark.table.Table" in type_name:
-            to_return = snowpark_mocks.Table()
-        elif "snowpark.dataframe.DataFrame" in type_name:
-            to_return = snowpark_mocks.DataFrame()
-        else:
-            to_return = (
-                pyspark_mocks.create_pyspark_dataframe_with_mocked_personal_data()
-            )
-
-        @st.experimental_memo
+    @parameterized.expand(
+        [
+            (SnowpandasSeries(pd.Series(np.random.randn(2))),),
```

Comment:
nit: alphabetical sorting helps humans to quickly search the list for coverage of a specific framework